src/undofeature.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import Feature from '../feature.js';
+import UndoCommand from './undocommand.js';
+
+/**
+ * Undo feature.
+ *
+ * Undo features brings in possibility to undo and re-do changes done in Tree Model by deltas through Batch API.
+ */
+export default class UndoFeature extends Feature {
```

Comment:
Will we call bold feature `Bold`, etc.? I fear for some name collisions and having a `Feature` suffix makes it clear what it is.